Sources/ContainerClient/Utility.swift

```diff
@@ -179,12 +179,16 @@ public struct Utility {
         config.mounts = resolvedMounts
 
         config.virtualization = management.virtualization
-
-        config.networks = try getAttachmentConfigurations(containerId: config.id, networkIds: management.networks)
-        for attachmentConfiguration in config.networks {
-            let network: NetworkState = try await ClientNetwork.get(id: attachmentConfiguration.network)
-            guard case .running(_, _) = network else {
-                throw ContainerizationError(.invalidState, message: "network \(attachmentConfiguration.network) is not running")
+        
+        if management.noNetwork {
+            config.networks = [AttachmentConfiguration(network: ClientNetwork.defaultNoNetworkName, options: AttachmentOptions(hostname: config.id))]
```

Comment:
Not sure if it would be appropriate to set the `config.networks` to an empty array or set add an attachment like so above

```suggestion
            config.networks = []
```